http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/ClusterToolMonitor.scala

```diff
@@ -88,21 +87,15 @@ class ClusterToolMonitor(
     )
 
   def checkClusterStatus(cluster: RunningCluster): IO[List[ToolStatus]] =
-    ClusterImageType.values.toList.traverseFilter { tool =>
-      ClusterContainerServiceType.imageTypeToClusterContainerServiceType.get(tool).traverse {
-        _.isProxyAvailable(cluster.googleProject, cluster.clusterName)
-          .map(status => {
-            //the if else is necessary because otherwise we will be reporting the metric 'welder down' on all clusters without welder, which is not the desired behavior
-            //TODO: change to  `ToolStatus(status, tool, cluster)` when data syncing is fully rolled out
-            if (!cluster.welderEnabled && tool == Welder) {
-              ToolStatus(true, tool, cluster)
-            } else {
-              ToolStatus(status, tool, cluster)
-            }
-          })
-      }
+    cluster.containers.traverse { tool =>
+      tool
+        .isProxyAvailable(cluster.googleProject, cluster.clusterName)
+        .map { status =>
+          ToolStatus(status, tool, cluster)
+        }
     }
-
 }
 
-final case class RunningCluster(googleProject: GoogleProject, clusterName: ClusterName, welderEnabled: Boolean)
+final case class RunningCluster(googleProject: GoogleProject,
```

Comment:
I was wondering if we want to move `RunningCluster` to `LeonardoModel` or something, since it's used outside of this class

Reply:
maybe to https://github.com/DataBiosphere/leonardo/blob/develop/core/src/main/scala/org.broadinstitute.dsde.workbench.leonardo/models.scala?

Reply:
Yeah that sounds good

Reply:
Actually that would require pulling more things into `models.scala`. Putting in `LeonardoModel` for now.. might be good to have a tech debt ticket to populate models.scala.